### PR TITLE
Gauges control

### PR DESCRIPTION
**Pull request by Macias89-SC**
Add speed gauge and fuel level gauge

### CODE REVIEW

**Comment by ziobron** [comment]
@Macias89-SC, we do not have your data in the Coders School database.
1. Please fill in this form https://forms.gle/rkzLsMUSBdnGQgdM9
2. Edit this PR to run this automation once again (you can just add a space in the description or title).
3. Wait one minute. If any label was added to this PR (like Standard, Premium, Podstawy C++, STL) then everything is fine :)
4. If your solution passed, but nothing happens, modify your code (eg. add an empty line) to run the workflow again.

If you don't know what to do - ask on Discord and provide a link to this PR.

File: content/MainBackground.ui.qml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+This is a UI file (.ui.qml) that is intended to be edited in Qt Design Studio only.
+It is supposed to be strictly declarative and only uses a subset of QML. If you edit
+this file manually, you might introduce QML code that is not supported by Qt Design Studio.
+Check out https://doc.qt.io/qtcreator/creator-quick-ui-forms.html for details on .ui.qml files.
+*/
+
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+
+Rectangle {
```
Comment (KrzucH) [inline content/MainBackground.ui.qml:11]: I think we will use same background to all Dashboard.

**Comment by ziobron** [comment]
Thanks @KrzucH for Code Review.
🏅 1 XP granted

File: content/SpeedGauge.ui.qml [hunk 2]
```diff
@@ -0,0 +1,346 @@
...
+        id: main
+        x: 0
+        y: 0
+        width: 500
+        height: 500
+        source: "SpeedGaugePicture/MainBackground.svg"
+        scale: 1
+        fillMode: Image.PreserveAspectFit
+
+        Text {
```
Comment (ziobron) [inline content/SpeedGauge.ui.qml:32]: Maybe it's worth to use a RowLayout? https://doc.qt.io/qt-6/qtquicklayouts-index.html

File: content/SpeedGauge.ui.qml [hunk 3]
```diff
@@ -0,0 +1,346 @@
...
+                frame: 81
+                value: "#5ca8ba"
+            }
+
+            Keyframe {
+                frame: 83
+                value: "#ffffff"
+            }
+        }
+        */
```
Comment (ziobron) [inline content/SpeedGauge.ui.qml:269]: to be removed?

File: content/SpeedGauge.ui.qml [hunk 4]
```diff
@@ -0,0 +1,346 @@
...
+        height: 39
+        color: "#ffffff"
+        text: qsTr("100")
+        font.pixelSize: 22
+        font.family: "Verdana"
+        font.bold: true
+    }
+
+    Text {
+        id: text8
```
Comment (ziobron) [inline content/SpeedGauge.ui.qml:286]: ids should be meaningful, like `gauge100Text`

File: content/SpeedGauge.ui.qml [hunk 5]
```diff
@@ -0,0 +1,346 @@
...
+    width: 500
+    height: 500
+    //property alias int_kmh
+
+    Image {
+        id: main
+        x: 0
+        y: 0
+        width: 500
+        height: 500
```
Comment (llisnikowski) [inline content/SpeedGauge.ui.qml:27]: Use `anchors.fill: parent`

File: content/SpeedGauge.ui.qml [hunk 6]
```diff
@@ -0,0 +1,346 @@
...
+import QtQuick.Timeline 1.0
+import QtQuick.Studio.LogicHelper 1.0
+
+Item {
+    id: speed
+    width: 500
+    height: 500
+    //property alias int_kmh
+
+    Image {
```
Comment (llisnikowski) [inline content/SpeedGauge.ui.qml:22]: You can use Image as main item

**Review by llisnikowski — CHANGES_REQUESTED** [review]

**Pull request closed without merging**